Content.Shared/GameObjects/Components/StandingStateComponent.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Players;
+using Robust.Shared.Serialization;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.GameObjects.Components
+{
+    [RegisterComponent]
+    public sealed class StandingStateComponent : Component
+    {
+        public override string Name => "StandingState";
+
+        public override uint? NetID => ContentNetIDs.STANDING_STATE;
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("downSoundCollection")]
+        public string? DownSoundCollection { get; } = "BodyFall";
+
+        [ViewVariables]
+        [DataField("standing")]
+        public bool Standing
+        {
+            get => _standing;
+            set
+            {
+                if (_standing == value) return;
+
+                _standing = value;
+                if (_standing)
+                {
+                    Owner.EntityManager.EventBus.RaiseLocalEvent(Owner.Uid, new StandEvent());
+                }
+                else
+                {
+                    Owner.EntityManager.EventBus.RaiseLocalEvent(Owner.Uid, new DownEvent());
+                }
+
+                Dirty();
```

Comment:
Either of these approaches still have the same ECS problem of access modifiers being shit because any system can modify this when they really shouldn't but I'll change it.